kalite/distributed/templates/distributed/learn.html

```diff
@@ -27,10 +27,10 @@
         {% compress css file perseuscss %}
         <link rel="stylesheet" type="text/css" href="{% static 'js/distributed/perseus/ke/css/khan-exercise.css' %}">
         <link rel="stylesheet" type="text/css" href="{% static 'css/distributed/video-js.css' %}">
-
         <link rel="stylesheet" type="text/css" href="{% static "js/distributed/perseus/lib/katex/katex.min.css" %}" />
         <link rel="stylesheet" type="text/css" href="{% static "js/distributed/perseus/lib/mathquill/mathquill.css" %}" />
         <link rel="stylesheet" type="text/css" href="{% static "js/distributed/perseus/build/perseus-3.css" %}" />
+        <link rel="stylesheet" type="text/css" href="{% static 'css/distributed/video-js-override.css' %}">
```

Comment:
This CSS file looks like a better place to put my focus changes from #4190. @rtibbles ?